src/Tabs/TabGroup.js

```diff
@@ -46,17 +46,18 @@ class TabGroup extends Component {
 
     // create tab list
     renderTabs = () => {
-        return React.Children.map(this.props.children, (child, index) => {
+        return React.Children.toArray(this.props.children).map((child, index) => {
             return this.cloneElement(child, index);
         });
     };
 
     // create content to show below tab list
     renderContent = () => {
-        return React.Children.map(this.props.children, (child, index) => {
+        return React.Children.toArray(this.props.children).map((child, index) => {
             return (
                 <TabContent
                     {...child.props.tabContentProps}
+                    key={child.props.id || index}
```

Comment:
Maybe just `key={index}` since the `id` prop is optional on each tab?

Reply:
I think using `key={child.props.id || index}` prevents an important bug.

Suppose someone is rendering tabs like this:
```js
const {tabs} = this.state;
return (
  <TabGroup>
    {tabs.map(tab => <Tab title={tab.title} />)}
  </TabGroup>
)
```
If tabs from state were to be updated to a different array, React may not render the change correctly if it is using `index` as a key.  Two different tabs would have the same key.

Writing it like `key={child.props.id || index}` catches the typical use case of static tabs, but allows for dynamic tabs if an id is passed through props.

```js
const {tabs} = this.state;
return (
  <TabGroup>
    {tabs.map(tab => <Tab id={tab.id} title={tab.title} />)}
  </TabGroup>
)
```

Though, yeah, that is a separate issue.  The propTypes of `TabContent` should be updated to include id as well, if this is approved.

Reply:
I'll try to create a broken example in the docs page to illustrate it.

Reply:
welp, I couldn't create any issues with using the index, so disregard.